src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -34,7 +33,7 @@
 namespace Microsoft.ML.Runtime.HalLearners
```

Comment:
>Microsoft.ML.Runtime.HalLearners [](start = 10, length = 32)

I'm surprised Pete didn't comment about namespace, especially what it's still "Runtime". But I guess we didn't start to change namespaces for learners yet.